lib/app/config/router.dart

```diff
@@ -60,6 +61,12 @@ final goRouter = GoRouter(
         Post post = state.extra as Post;
         return PostDetailView(post);
       },
-    )
+    ),
+    GoRoute(
+      path: '/settings',
+      name: 'settings',
```

Comment:
it's. a good patteren pattern of putting the path and/or name as a static variable in the widget itself (or a static list of Routes, but i lke in the widget itself) so you know how to pushNamed to it
ie SettingsView.name or SettingsView.path and it's not a "magic" string" yo have to make sure is spelled right, has the right casing, login vs signin etc.